users/signals.py

```diff
@@ -22,7 +22,7 @@ def create_profile(sender, instance, created, **kwargs):
                                role=Role.objects.get(name="Trainee"))
 
 
-@receiver(post_save, sender=User)
+@receiver(post_save, sender=CustomUser)
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import